src/org/dotnet/ci/pipelines/Pipeline.groovy

```diff
@@ -347,13 +347,17 @@ class Pipeline {
     //  parameter - Parameter set to run the pipeline with
     // Returns
     //  Newly created pipeline job
-    public def triggerPipelineOnEvent(TriggerBuilder triggerBuilder, Map<String,Object> params = [:]) {
+    public def triggerPipelineOnEvent(TriggerBuilder triggerBuilder, Map<String,Object> params = [:], String jobName = null) {
         // Determine the job name
         // Job name is based off the parameters 
 
         def isPR = triggerBuilder.isPRTrigger()
-        def jobName = getPipelineJobName(params)
-        def fullJobName = Utilities.getFullJobName(jobName, isPR)
+        def _jobName = jobName
+        def fullJobName = jobName
+        if(_jobName == null) {
+            _jobName = Pipeline.getPipelineJobName(_baseJobName, params)
+            fullJobName = Utilities.getFullJobName(_jobName, isPR)
```

Comment:
Any reason we aren't calling getFullJobName in the case of an explicit job name?  It is just the thing that adds _prtest in the case of pr.  Wouldn't that mean an overlap of job names between trigger methods?